datafusion/physical-plan/Cargo.toml

```diff
@@ -69,6 +68,7 @@ rand = { workspace = true }
 tokio = { workspace = true }
 
 [dev-dependencies]
+datafusion-functions-aggregate = { workspace = true }
```

Comment:
👍 nice

As a follow on it would be great to remove this dependency too, but having it as a dev dependency is clearly better than a real dependency in my mind

Reply:
Maybe we can move that test into the `sql_integration` binary or something

Reply:
I found that some tests call private methods, which means we can't move them to a different package. Perhaps we can address this in a subsequent PR.